op-node/cmd/main.go

```diff
@@ -53,13 +55,22 @@ func main() {
 	)
 
 	app := cli.NewApp()
-	app.Flags = flags.Flags
 	app.Version = VersionWithMeta
 	app.Name = "opnode"
 	app.Usage = "Optimism Rollup Node"
 	app.Description = "The deposit only rollup node drives the L2 execution engine based on L1 deposits."
+	app.Commands = []cli.Command{
+		{
+			Name:        "p2p",
+			Subcommands: p2p.Subcommands,
+		},
+		{
+			Name:   "start",
```

Comment:
k8s sister PR?

Reply:
Once this is merged, yes.

Reply:
Does start really have to be a sub-command? I would like to run the `RollupNodeMain` as the main command by default. Same thing geth does.